.gitignore

```diff
@@ -35,7 +35,5 @@
 build/
 # ignore the packaging folder contents from git
 packaging/
-# graph config is automatically setup, so we can ignore this
-include/CXXGraphConfig.h
```

Comment:
Removed so that we don't depend on generated files during configure time.

Reply:
OK